### PR TITLE
Docs: authorization.md, fixed typo

### PR DESCRIPTION
**Pull request by sammyskills**

### CODE REVIEW

File: docs/authorization.md [hunk 1]
```diff
@@ -6,20 +6,20 @@
...
-      - [removeGroup()](#removegroup)
-      - [syncGroups()](#syncgroups)
-      - [getGroups()](#getgroups)
+    - [addGroup()](#addgroup)
+    - [removeGroup()](#removegroup)
+    - [syncGroups()](#syncgroups)
+    - [getGroups()](#getgroups)
   - [User Activation](#user-activation)
     - [Checking Activation Status](#checking-activation-status)
   - [Activating a User](#activating-a-user)
```
Comment (datamweb) [inline docs/authorization.md:25]:
```suggestion
    - [Activating a User](#activating-a-user)
```
Also change for `Deactivating a User`.

File: docs/authorization.md [hunk 2]
```diff
@@ -264,7 +264,7 @@ if ($user->isNotActivated()) {
 
 ## Activating a User
```
Comment (datamweb) [inline docs/authorization.md:265]:
```suggestion
### Activating a User
```
Also change for `Deactivating a User`.

File: docs/authorization.md (hunk 1 above)
Comment (sammyskills) [reply docs/authorization.md]: Done.

File: docs/authorization.md (hunk 2 above)
Comment (sammyskills) [reply docs/authorization.md]: Done.

**Review by datamweb — APPROVED** [review]
Thank you!

**Pull request merged**